.sailsrc

```diff
@@ -7,6 +7,9 @@
     "sails-generate": "1.16.13"
   },
   "hooks": {
-    "grunt": false
+    "grunt": false,
+    "orm": false,
```

Comment:
Question: What does this disable? Is this what turns off Waterline?

Reply:
yes that's my understanding, this was me basically copying from the instructions in the repo for the Sequelize Hook.